src/ethereum/api/collection.ts

```diff
@@ -184,9 +184,14 @@ export class CollectionAPI extends BaseGraphAPI {
   fetchCollections = async (
     filters: CollectionQueryFilters
   ): Promise<CollectionFragment[]> => {
-    return this.paginate(['collections'], {
-      query: getCollectionsQuery(filters),
-    })
+    return logExecutionTime(
+      () =>
+        this.paginate(['collections'], {
+          query: getCollectionsQuery(filters),
+        }),
+      this.logger,
+      'Collections fetch'
+    )
```

Comment:
Do we want to keep logging the time?

Reply:
not really, we're already logging the graph's queries timeout. I'll remove this